lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -284,6 +295,27 @@ def get_target_from_rule(module, ec2, rule, name, group, groups, vpc_id):
     return group_id, ip, target_group_created
 
 
+def ensure_tags(ec2, group, tags, changed, module):
+    try:
+        resource_id = group.id
+        cur_tags = group.tags
+        if cur_tags == tags:
```

Comment:
Hmm, should we explicitly convert the new tag values to strings, or is there a way to use the module parameter API to tell Ansible that we want a hash of strings and let it do the conversion for us?